siriuspy/siriuspy/devices/bpm.py

```diff
@@ -1351,7 +1290,7 @@ def mturn_wait_update_timestamps(
             int: code describing what happened:
                 -1: initial timestamps were not defined;
                 =0: data updated.
-                >0: index of the first FOFB controller which did not update
+                >0: index of the first BPM which did not update
                     plus 1.
```

Comment:
I think this text fits in the previous line, doesn't it?

Reply:
done!